floodfill_selection/volume_selection.py

```diff
@@ -15,13 +15,12 @@
 from PyQt4 import QtGui, QtCore
 from vispy import app, scene
 from matplotlib import path
-import pyfits
 from astropy.io import fits
 from astropy.utils.data import get_pkg_data_filename
 
 from multivol import MultiVolume
 from multivol import get_translucent_cmap
-import flood_fill_3d
+from floodfill_scipy import floodfill_scipy
```

Comment:
I've switched to using my version of the algorithm for now since it doesn't require recompiling